src/frontend/packages/core/src/jetstream.helpers.ts

```diff
@@ -1,58 +1,14 @@
 import { HttpErrorResponse } from '@angular/common/http';
 
-/**
- * Actual error response from stratos
- */
-export interface JetStreamErrorResponse<T = any> {
-  error: {
-    status: string;
-    statusCode: number;
-  };
-  /**
-   * Actual response from proxied endpoint
-   */
-  errorResponse: T;
-}
+import { isHttpErrorResponse, JetStreamErrorResponse } from '../../store/src/jetstream';
 
-export function isJetstreamError(err: any): JetStreamErrorResponse {
-  return !!(
-    err &&
-    err.error &&
-    err.error.status &&
-    err.error.statusCode &&
-    'errorResponse' in err
-  ) ? err as JetStreamErrorResponse : null;
-}
-
-// TODO It would be nice if the BE could return a unique para for us to check for. #3827
-// There is always a chance that this will return a false positive (more so with extensions).
-export function hasJetStreamError(pages: Partial<JetStreamErrorResponse>[]): JetStreamErrorResponse {
-  if (!pages || !pages.length) {
-    return null;
-  }
-  return pages.find(page => {
-    return isJetstreamError(page);
-  }) as JetStreamErrorResponse;
-}
 
 export function jetStreamErrorResponseToSafeString(response: JetStreamErrorResponse): string {
```

Comment:
I think we should move this too, purely so all the jetsream error handlers are in the same place (function is only used in one file in cf package)